csrc/index_compute.cpp

```diff
@@ -1979,11 +1984,14 @@ std::vector<Val*> Index::getStrides(TensorView* tv) {
     for (const auto i : c10::irange(alloc_dom.size())) {
       if (alloc_dom[i]->isReduction() || alloc_dom[i]->isStride()) {
```

Comment:
Same here, correct me if I'm wrong but I think that a one-line change does the job
```suggestion
      if (alloc_dom[i]->isReduction() || alloc_dom[i]->isStride() || alloc_dom[i]->isDeviceDim()) {
```